src/pytest_replay/__init__.py

```diff
@@ -39,6 +44,24 @@ def pytest_addoption(parser):
     )
 
 
+@dataclasses.dataclass
+class ReplayTestMetadata:
```

Comment:
This class is not only responsible for the metadata, but everything else related to the plugin, like start, outcome, etc. To avoid confusing it with the "metadata" attribute, how about naming it differently, say:

```suggestion
class ReplayTestInfo:
```

Reply:
I agree, ``ReplayTestInfo`` is better, indeed